templates/compose/hatchet.yaml

```
@@ -0,0 +1,106 @@
+# documentation: https://docs.hatchet.run/self-hosting/docker-compose
+# slogan: Hatchet is a high-throughput, low-latency computing service. It's built on an open-source, fault-tolerant queue, allowing work to be delivered as fast as your system can handle
+# tags: ai-agents,background-tasks,data-pipelines,scheduling
+# logo: svgs/hatchet.svg
+# port: 80
+
+services:
+  postgres:
+    image: postgres:16-alpine
+    volumes:
+      - postgresql-data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=$SERVICE_USER_POSTGRES
+      - POSTGRES_PASSWORD=$SERVICE_PASSWORD_POSTGRES
+      - POSTGRES_DB=${POSTGRES_DB:-hatchet}
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]
+      interval: 5s
+      timeout: 20s
+      retries: 10
+  rabbitmq:
+    image: rabbitmq:3-management
+    hostname: "rabbitmq"
+    environment:
+      - SERVICE_URL_RABBITMQ_15672
+      - RABBITMQ_DEFAULT_USER=$SERVICE_USER_RABBITMQ
+      - RABBITMQ_DEFAULT_PASS=$SERVICE_PASSWORD_RABBITMQ
+      - PORT=${RABBITMQ_PORT:-5672}
+    healthcheck:
+      test: rabbitmq-diagnostics -q ping
+      interval: 5s
+      timeout: 30s
+      retries: 10
+    volumes:
+      - rabbitmq-data:/var/lib/rabbitmq/
+  migration:
+    image: ghcr.io/hatchet-dev/hatchet/hatchet-migrate:latest
+    command: /hatchet/hatchet-migrate
+    restart: no
```

Adding `exclude_from_hc: true` will prevent this migration service affecting the service health status
```suggestion
    restart: no
    exclude_from_hc: true
```